cfme/utils/targeted_refresh.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+"""
+class for targeted refresh tests
+"""
+
+
+class TargetedRefresh(object):
+
+    def __init__(self):
+        self._check_strings = {}
+        self._entities = []
+        self._triggered_refreshes = []
+
+    @property
+    def entities(self):
```

Comment:
**required** those properties offer no value in python, please use direct attributes

Reply:
Done.